hil/commands/admin.py

```diff
@@ -1,16 +1,96 @@
 """Implement the hil-admin command."""
-from hil import config, model
+from hil import config, model, deferred, server, migrations, rest
 from hil.commands import db
 from hil.commands.migrate_ipmi_info import MigrateIpmiInfo
 from hil.commands.util import ensure_not_root
 from hil.flaskapp import app
+
+from time import sleep
 from flask.ext.script import Manager
 
+import sys
+import logging
+from click import IntRange
+
 manager = Manager(app)
+
 manager.add_command('db', db.command)
 manager.add_command('migrate-ipmi-info', MigrateIpmiInfo())
 
 
+@manager.command
+def serve_networks():
+    """Start the HIL networking server"""
+    logger = logging.getLogger(__name__)
+    server.init()
+    server.register_drivers()
+    server.validate_state()
+    migrations.check_db_schema()
+
+    # Check if config contains usable sleep_time
+    if (config.cfg.has_section('network-daemon') and
+            config.cfg.has_option('network-daemon', 'sleep_time')):
+        try:
+            sleep_time = config.cfg.getfloat('network-daemon', 'sleep_time')
+        except (ValueError):
+            sys.exit("Error: sleep_time set to non-float value")
+        if sleep_time <= 0 or sleep_time >= 3600:
+            sys.exit("Error: sleep_time not within bounds "
+                     "0 < sleep_time < 3600")
+        if sleep_time > 60:
+            logger.warn('sleep_time greater than 1 minute.')
+    else:
+        sleep_time = 2
+
+    while True:
+        # Empty the journal until it's empty; then delay so we don't tight
+        # loop.
+        while deferred.apply_networking():
+            pass
+        sleep(sleep_time)
+
+
+@manager.option('-p', '--port', type=IntRange(0, 2**16-1), default=5000)
+def run_dev_server(port):
```

Comment:
Are we still making `-` our default delimiter for commands? If so, we might as well remove the underscores.

Reply:
Again, this is a Python variable name.

Reply:
Ah, right, this also ends up being the name of the command. Yeah, let's override this if we can.